Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/CwaUserCard.kt

```diff
@@ -41,9 +43,9 @@ class CwaUserCard(parent: ViewGroup) :
 
     private fun formatBirthDate(dateOfBirthFormatted: String): String =
         try {
-            dateOfBirthFormatted.split("-").reversed().joinToString(separator = ".")
+            dateOfBirthFormatted.parseServerFormat().toDayFormat()
```

Comment:
Birthdate here has special cases, birthdate can have the following cases:
- yyyy only ex: 1990
- yyyy-MM ex: 1990-10
- yyyy-MM-dd ex: 1990-10-01
So using standard formatter won't work as expected

Reply:
@marcauberer  FYI ☝️ ?

Reply:
Should be fixed now 👍🏼